OpenRA.Mods.Common/Widgets/Logic/Editor/ActorSelectorLogic.cs

```diff
@@ -202,6 +203,18 @@ public ActorSelectorLogic(Widget widget, World world, WorldRenderer worldRendere
 			InitializeActorPreviews();
 		}
 
+		protected override void Dispose(bool disposing)
+		{
+			editorLayer.PlayerRemoved -= EditorLayer_PlayerRemoved;
+		}
+
+		void EditorLayer_PlayerRemoved(object sender, PlayerRemovedEventArgs e)
```

Comment:
The underscore notation looks a bit alien here.

Reply:
can fix, no probs!